### PR TITLE
Mobile view redesign#403/zubkov

### PR DESCRIPTION
**Pull request by olegz7**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,5 +1,113 @@
+
 <div class="app-wrapper">
-  <app-header></app-header>
-  <app-shell></app-shell>
-  <app-footer></app-footer>
+  <!-- Content of the page with opened sidebar -->
+  <mat-drawer-container>
```
Comment (dmitryy90) [inline src/app/app.component.html:4]: extract it to component

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,10 +1,103 @@
...
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+
+
+export class AppComponent implements OnInit, OnDestroy{
+
+  readonly Languages: typeof Languages = Languages;
+  selectedLanguage: string = 'uk'
```
Comment (dmitryy90) [inline src/app/app.component.ts:33]: selected lang should be get from storage

File: src/app/header/header.component.scss [hunk 3]
```diff
@@ -338,22 +352,24 @@
       justify-content: space-between;
     }
     &_top {
+      display: flex;
       flex-direction: row;
       align-items: center;
-      position: fixed;
+      position: absolute;
```
Comment (dmitryy90) [inline src/app/header/header.component.scss:358]: why do you change position to absolute?

File: src/app/shared/store/navigation.state.ts [hunk 4]
```diff
@@ -1,29 +1,34 @@
...
+  static navigationPaths(state: NavStateModel): Navigation[] { return state.navigation }
+
+  @Selector()
+  static sidenavOpenTrue(state: NavStateModel): boolean {return state.sidenavOpen}
 
   @Selector()
   static navigationPathsMobile(state: NavStateModel): Navigation[] {
      const navigation = [...state.navigation]
-     return [navigation.pop()] 
+     return [navigation.pop()]
```
Comment (dmitryy90) [inline src/app/shared/store/navigation.state.ts:31]: please put semicolon where they are missed

File: src/app/shared/store/navigation.state.ts [hunk 5]
```diff
@@ -48,4 +53,15 @@ export class NavigationState {
       navigation: []
     })
   }
-}
\ No newline at end of file
+
+  @Action(SidenavToggle)
+  SidenavToggle({patchState,getState}:StateContext<NavStateModel>): void {
+    const sidenavOpenState = getState().sidenavOpen
+        patchState({
```
Comment (dmitryy90) [inline src/app/shared/store/navigation.state.ts:60]: 2 spaces tab

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/header/header.component.scss (hunk 3 above)
Comment (olegz7) [reply src/app/header/header.component.scss:358]: returned, fixed and z-index as well

File: src/app/shared/store/navigation.state.ts (hunk 4 above)
Comment (olegz7) [reply src/app/shared/store/navigation.state.ts:31]: don't know why it showed up here, but it's not my changes
anyway, semicolon added

File: src/app/shared/store/navigation.state.ts (hunk 5 above)
Comment (olegz7) [reply src/app/shared/store/navigation.state.ts:60]: fixed

File: src/app/app.component.html (hunk 1 above)
Comment (olegz7) [reply src/app/app.component.html:4]: extracted

File: src/app/app.component.ts (hunk 2 above)
Comment (olegz7) [reply src/app/app.component.ts:33]: please let me know when you could clarify about this question

**Review by dmitryy90 — APPROVED** [review]

**Pull request merged**